CorsixTH/com.github.CorsixTH.CorsixTH.appdata.xml

```diff
@@ -1,16 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
 <component type="desktop">
   <id>corsix-th.desktop</id>
+  <!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
   <metadata_license>CC0</metadata_license>
-  <project_license>GPL-3.0+</project_license>
```

Comment:
Sorry, I misspoke, our code is just MIT. Does the AppData license need to cover dependencies like ffmpeg as well though?

Reply:
Though, the random.c code is BSD-3 clause... ok. 😄